doc/nrf/ug_radio_fem.rst

```diff
@@ -94,10 +146,96 @@ To use nRF21540 in GPIO mode, complete the following steps:
+
+1. Add the following node in the devicetree file:
+
+   .. code-block::
+
+      / {
+            nrf_radio_fem: name_of_fem_node {
+               compatible  = "nordic,nrf21540-fem";
+               tx-en-gpios = <&gpio0 13 GPIO_ACTIVE_HIGH>;
+               rx-en-gpios = <&gpio0 14 GPIO_ACTIVE_HIGH>;
+               pdn-gpios   = <&gpio0 15 GPIO_ACTIVE_HIGH>;
+               spi-if = <&nrf_radio_fem_spi>
+         };
+      };
+#. Optionally replace the SPI bus device name ``nrf_radio_fem_spi``.
+#. Replace the pin numbers provided for each of the required properties:
+
+   * ``tx-en-gpios`` - GPIO characteristic of the device that controls the ``TX_EN`` signal of nRF21540.
+   * ``rx-en-gpios`` - GPIO characteristic of the device that controls the ``RX_EN`` signal of nRF21540.
+   * ``pdn-gpios`` - GPIO characteristic of the device that controls the ``PDN`` signal of nRF21540.
+
+   These properties correspond to ``TX_EN``, ``RX_EN``, and ``PDN`` pins of nRF21540 that are supported by software FEM.
+
+   The``phandle-array`` type is commonly used for describing GPIO signals in Zephyr's devicetree.
+   The first element ``&gpio0`` refers to the GPIO port ("port 0" has been selected in the example shown).
+   The second element is the pin number on that port.
+   The last element must be ``GPIO_ACTIVE_HIGH`` for nRF21540, but for a different FEM module you can use ``GPIO_ACTIVE_LOW``.
+
+   Set the state of the remaining control pins according to the `nRF21540 Product Specification`_.
+#. Add a following SPI bus device node on the devicetree file:
+
+   .. code-block::
+
+      fem_spi: &spi3 {
+	      status = "okay";
+	      sck-pin = <47>;
+	      miso-pin = <46>;
+	      mosi-pin = <45>;
+	      cs-gpios = <&gpio0 21 GPIO_ACTIVE_LOW>;
+
+	      nrf_radio_fem_spi: nrf21540_fem_spi@0 {
+		      compatible = "nordic,nrf21540-fem-spi";
+		      status = "okay";
+		      reg = <0>;
+		      label = "FEM_SPI_IF";
+		      spi-max-frequency = <8000000>;
+	      };
+      };
+
+   In this example, the nRF21540 is controlled by the ``spi3`` bus.
+   Replace the SPI bus according to your hardware design.
+   Replace the SPI bus device name ``nrf_radio_fem_spi`` with the name from the previous step.
+
+#. Replace the pin numbers provided for each of the required properties:
+
+   * ``sck-pin`` -  GPIO pin number of the device that controls the ``SCK`` signal of the nRF21540.
+   * ``miso-pin`` - GPIO pin number of the device that controls the ``MISO`` signal of the nRF21540.
+   * ``mosi-pin`` - GPIO pin number of the device that controls the ``MOSI`` signal of the nRF21540.
+   * ``cs-gpio`` - GPIO characteristic of the device that controls the ``CSN`` signal of nRF21540.
+
+   ``sck-pin``, ``miso-pin``, ``mosi-pin`` are absolute pin numbers.
+   Use the following formula to calculate them::
+
+      pin_no = b\*32 + a
+
+   In this formula ``a`` and ``b`` are from the Pin value in the table (Pb.a).
```

Comment:
Can we say that 'b' is a port number and 'a' ia a pin number within the port. Which table is referred to here?

Reply:
I wondered the same about the table.

Reply:
You a right. I changed it :)